build.sbt

```
@@ -91,7 +91,11 @@ lazy val tests = crossProject(JVMPlatform, JSPlatform, NativePlatform)
         toolkitTest.native / publishLocal
       )
       .value,
-    buildInfoKeys += "platform" -> "native"
+    buildInfoKeys ++= List(
+      "platform" -> "native",
+      // Remember to remove it once the whole stack is ported to SN 0.5.x
+      "nativeVersion" -> "0.4.17"
```

```suggestion
      "nativeVersion" -> nativeVersion
```

https://github.com/scala-native/scala-native/blob/8302845c12f1cec7110d1f809fc238afc1121239/sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala#L15

---

👆 smart